survival_guide/3_operations.adoc

```diff
@@ -677,39 +701,51 @@ With a access/secret pair, you’ll need to add the secret:
 [source,yaml]
 ----
 s3:
-    enableS3: true
-    enableIAM: false
-    accessKeyName: aws-access-key
-    secretKeyName: aws-secret-key
-    logDirectory: s3a://[BUCKET-NAME]
+  enableS3: true
+  enableIAM: false
+  accessKeyName: aws-access-key
+  secretKeyName: aws-secret-key
+  logDirectory: s3a://[BUCKET_NAME]
 ----
 
 ===== Configuring Spark
 
-After the History Server has been set up, then the Fusion job-launcher deployment ConfigMap’s `application.yaml` key will need to be updated with these Spark settings so the driver and executors know where to write out their logs. In this example, we’re redirecting to a GCS bucket:
+After starting the Spark History Server, we must update the config map for Fusion's job-launcher so it can write logs to the same location that Spark History Server is reading from.
+
+In this example, having installed Fusion into a namespace of `sparkhistory`, we will edit the config map to write the logs to the same Google Cloud Storage bucket we configured the Spark History Server to read from.
+
+[source,bash]
+----
+kubectl edit cm sparkhistory-job-launcher
+----
+
+Update the `spark` key with these settings and then restart the `job-launcher` pod.
+
 
 [source,yaml]
 ----
 spark:
-    eventLog:
-        enabled: true
-        compress: true
-        dir: gs://[BUCKET-NAME]
-    hadoop:
-        google:
-            cloud:
-                auth:
-                    service:
-                        account:
-                            json:
-                                keyfile: /etc/secrets/[SECRET_KEY_NAME].json]
-    kubernetes:
-        driver:
-            secrets:
-                history-secrets: /etc/secrets
-        executor:
-            secrets:
-                history-secrets: /etc/secrets
+  …
+  eventLog:
+    enabled: true
+    compress: true
+    dir: gs://[BUCKET_NAME]
+  hadoop:
+    google:
+      cloud:
+        auth:
+          service:
+            account:
+              json:
+                keyfile: /etc/secrets/[ACCOUNT_NAME].json
+  kubernetes:
+    driver:
+      secrets:
+        history-secrets: /etc/secrets
```

Comment:
pretty sure using /etc/secrets conflicts with some other secret (service account key?)
I think you need to use /etc/history-secrets or something unique instead for all of these instead of /etc/secrets